.github/workflows/build.yml

```diff
@@ -339,10 +339,16 @@ jobs:
       if: github.event_name == 'push'
       shell: bash
       run: >
+        if [[ "${GITHUB_REF}" =~ ^refs/tags/.* ]];
+        then
+          export DEPLOY_PATH='mixxx-{git_describe}/mixxx-{git_describe}-{package_slug}{ext}';
```

Comment:
The directory path code is repeated below for the manifest path. How about factoring that out to an environment variable used by both GH Actions steps?

Reply:
Not possible, the other code runs in a fresh VM.

Reply:
okay